src/pgmigrations/1693317600000_create_worlds_table.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationBuilder, PgType } from 'node-pg-migrate'
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createTable('worlds', {
+    name: { type: PgType.VARCHAR, notNull: true, primaryKey: true },
+    owner: { type: PgType.VARCHAR, notNull: false },
+    deployer: { type: PgType.VARCHAR, notNull: false },
```

Comment:
Same doubt here (because of the notNull)

Reply:
deployer could be null, yes. Say you buy a name, you don't deploy any scene, and you give some other wallet permission to deploy on your behalf. Then the row will exist in the DB (to store the ACL) but it doesn't have any deployments yet.